charts/argo-applicationset/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v2
```

Comment:
I used helm3 to generate this chart. Should it be `v1`?

Reply:
Helm 2 is deprecated, we can stay with v2